### DISCUSSION TITLE
Welcome to ChaiScript Discussions!

### DISCUSSION THREAD

**Discussion by lefticus** [ChaiScript/ChaiScript#567] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Avienneau** [comment]
Hey!

Thanks so much for enabling discussion around Chaiscript. I'm Aaron, I'm a consumer of Chaiscript and I absolutely love how easy it is to integrate Chaiscript into projects and how easy it is to use! I'm attempting to use Chaiscript as an embedded scripting language for my text-based game engine (MUD) written in C++, more on that to come!

**Comment by totalgee** [comment]
Hi, I'm Glen and I've been using ChaiScript for various (personal and work) projects for about six years, and have contributed a number of fixes over that time.

**Comment by BerndAmend** [comment]
Hi, thanks for the great project I have been using it for the last 5 years mostly at work and sometimes for my private projects. I'm pretty happy about the current state. Sadly the Website and Issues don't reflect this. I would love if these would get fixed. I'm also interested in helping out.

**Reply by lefticus** [reply]
The website is also hosted on github, I would gladly welcome contributions. I've had to choose how to allocate my time, and ChaiScript became lower priority. I love that people are maintaining the C++17 work and I might take a look at it sometime in the foreseeable future.

**Reply by totalgee** [reply]
@lefticus It's exciting and encouraging to see the recent flurry of activity here, thanks to you and your pal @leftibot ! Any chance you plan to talk (make a video) or write about the setup you're using to do it (I assume with Claude Code), which allows you to automate a lot of the work here to close issues and make fixes and PRs?

**Reply by lefticus** [reply]
I've recorded a few episodes about how to use AI safely, but now I'm afraid they'll be partially out of date by the time they air. I'm not sure how/if/when I'll present what I'm doing.  I know it's not very unique tho.

Basically I have a script that runs and gets any issue that I personally @ leftibot in, then hands those off to a Claude Code fresh instance. It's all in VMs and leftibot and the script only have access to public repositories and can only push to any fork that it makes. Then I'm personally reviewing it and push back if necessary on what the bot did.
